<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>794</width>
    <height>581</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>1</number>
    </property>
    <property name="topMargin">
     <number>1</number>
    </property>
    <property name="rightMargin">
     <number>1</number>
    </property>
    <property name="bottomMargin">
     <number>1</number>
    </property>
    <item>
     <widget class="QPlainTextEdit" name="textEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>794</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="actionnew"/>
    <addaction name="actionopen"/>
    <addaction name="actionsave"/>
    <addaction name="actionsaveas"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_E">
    <property name="title">
     <string>编辑(&amp;E)</string>
    </property>
    <addaction name="actionundo"/>
    <addaction name="actionredo"/>
    <addaction name="actioncut"/>
    <addaction name="actioncopy"/>
    <addaction name="actionpaste"/>
    <addaction name="actionfind"/>
    <addaction name="actionreplace"/>
    <addaction name="actionSelectAll"/>
   </widget>
   <widget class="QMenu" name="menu_G">
    <property name="title">
     <string>格式(&amp;G)</string>
    </property>
    <addaction name="actionLineWrap"/>
    <addaction name="actionsetnumber"/>
    <addaction name="actionfont"/>
    <addaction name="actionfontcolor"/>
    <addaction name="actionfontbackgroundcolor"/>
    <addaction name="actionebackgroundcolor"/>
   </widget>
   <widget class="QMenu" name="menu_V">
    <property name="title">
     <string>查看(&amp;V)</string>
    </property>
    <addaction name="actionShowToolBar"/>
    <addaction name="actionShowStatusBar"/>
   </widget>
   <widget class="QMenu" name="menu_H">
    <property name="title">
     <string>帮助(&amp;H)</string>
    </property>
    <addaction name="actionabout"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_E"/>
   <addaction name="menu_G"/>
   <addaction name="menu_V"/>
   <addaction name="menu_H"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionnew"/>
   <addaction name="actionopen"/>
   <addaction name="actionsave"/>
   <addaction name="actionsaveas"/>
   <addaction name="actionundo"/>
   <addaction name="actionredo"/>
   <addaction name="actioncut"/>
   <addaction name="actioncopy"/>
   <addaction name="actionpaste"/>
   <addaction name="actionfind"/>
   <addaction name="actionreplace"/>
   <addaction name="actionLineWrap"/>
   <addaction name="actionsetnumber"/>
   <addaction name="actionfont"/>
   <addaction name="actionShowToolBar"/>
   <addaction name="actionShowStatusBar"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>true</bool>
   </attribute>
  </widget>
  <action name="actionnew">
   <property name="icon">
    <iconset resource="notepad.qrc">
     <normaloff>:/img/open_in_new_FILL0_wght400_GRAD0_opsz48 - 副本.png</normaloff>:/img/open_in_new_FILL0_wght400_GRAD0_opsz48 - 副本.png</iconset>
   </property>
   <property name="text">
    <string>新建(&amp;N)</string>
   </property>
   <property name="toolTip">
    <string>新建</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionopen">
   <property name="icon">
    <iconset resource="notepad.qrc">
     <normaloff>:/img/expand_content_FILL0_wght400_GRAD0_opsz48.png</normaloff>:/img/expand_content_FILL0_wght400_GRAD0_opsz48.png</iconset>
   </property>
   <property name="text">
    <string>打开(&amp;O)</string>
   </property>
   <property name="toolTip">
    <string>打开</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionsaveas">
   <property name="icon">
    <iconset resource="notepad.qrc">
     <normaloff>:/img/save_as_FILL0_wght400_GRAD0_opsz48.png</normaloff>:/img/save_as_FILL0_wght400_GRAD0_opsz48.png</iconset>
   </property>
   <property name="text">
    <string>另存为(&amp;A)</string>
   </property>
   <property name="toolTip">
    <string>另存为</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionsave">
   <property name="icon">
    <iconset resource="notepad.qrc">
     <normaloff>:/img/save_FILL0_wght400_GRAD0_opsz48.png</normaloff>:/img/save_FILL0_wght400_GRAD0_opsz48.png</iconset>
   </property>
   <property name="text">
    <string>保存(&amp;S)</string>
   </property>
   <property name="toolTip">
    <string>保存</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionundo">
   <property name="icon">
    <iconset resource="notepad.qrc">
     <normaloff>:/img/undo_FILL0_wght400_GRAD0_opsz48.png</normaloff>:/img/undo_FILL0_wght400_GRAD0_opsz48.png</iconset>
   </property>
   <property name="text">
    <string>撤销(&amp;U)</string>
   </property>
   <property name="toolTip">
    <string>撤销</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionredo">
   <property name="icon">
    <iconset resource="notepad.qrc">
     <normaloff>:/img/forward_FILL0_wght400_GRAD0_opsz48 - 副本.png</normaloff>:/img/forward_FILL0_wght400_GRAD0_opsz48 - 副本.png</iconset>
   </property>
   <property name="text">
    <string>恢复</string>
   </property>
   <property name="toolTip">
    <string>恢复</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Z</string>
   </property>
  </action>
  <action name="actioncut">
   <property name="icon">
    <iconset resource="notepad.qrc">
     <normaloff>:/img/cut_FILL0_wght400_GRAD0_opsz48 - 副本.png</normaloff>:/img/cut_FILL0_wght400_GRAD0_opsz48 - 副本.png</iconset>
   </property>
   <property name="text">
    <string>剪切(&amp;T)</string>
   </property>
   <property name="toolTip">
    <string>剪切</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actioncopy">
   <property name="icon">
    <iconset resource="notepad.qrc">
     <normaloff>:/img/content_copy_FILL0_wght400_GRAD0_opsz48 - 副本.png</normaloff>:/img/content_copy_FILL0_wght400_GRAD0_opsz48 - 副本.png</iconset>
   </property>
   <property name="text">
    <string>复制(&amp;C)</string>
   </property>
   <property name="toolTip">
    <string>复制</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionpaste">
   <property name="icon">
    <iconset resource="notepad.qrc">
     <normaloff>:/img/content_paste_FILL0_wght400_GRAD0_opsz48 - 副本.png</normaloff>:/img/content_paste_FILL0_wght400_GRAD0_opsz48 - 副本.png</iconset>
   </property>
   <property name="text">
    <string>粘贴(&amp;V)</string>
   </property>
   <property name="toolTip">
    <string>粘贴</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionfind">
   <property name="icon">
    <iconset resource="notepad.qrc">
     <normaloff>:/img/find_replace_FILL0_wght400_GRAD0_opsz48 - 副本.png</normaloff>:/img/find_replace_FILL0_wght400_GRAD0_opsz48 - 副本.png</iconset>
   </property>
   <property name="text">
    <string>查找(&amp;F)</string>
   </property>
   <property name="toolTip">
    <string>查找</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionreplace">
   <property name="icon">
    <iconset resource="notepad.qrc">
     <normaloff>:/img/published_with_changes_FILL0_wght400_GRAD0_opsz48.png</normaloff>:/img/published_with_changes_FILL0_wght400_GRAD0_opsz48.png</iconset>
   </property>
   <property name="text">
    <string>替换(&amp;R)</string>
   </property>
   <property name="toolTip">
    <string>替换</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="actionLineWrap">
   <property name="icon">
    <iconset resource="notepad.qrc">
     <normaloff>:/img/backup_table_FILL0_wght400_GRAD0_opsz48 - 副本.png</normaloff>:/img/backup_table_FILL0_wght400_GRAD0_opsz48 - 副本.png</iconset>
   </property>
   <property name="text">
    <string>自动换行(&amp;W)</string>
   </property>
   <property name="toolTip">
    <string>自动换行</string>
   </property>
  </action>
  <action name="actionsetnumber">
   <property name="icon">
    <iconset resource="notepad.qrc">
     <normaloff>:/img/format_list_numbered_rtl_FILL0_wght400_GRAD0_opsz48.png</normaloff>:/img/format_list_numbered_rtl_FILL0_wght400_GRAD0_opsz48.png</iconset>
   </property>
   <property name="text">
    <string>显示行号(&amp;L)</string>
   </property>
   <property name="toolTip">
    <string>显示行号</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionfont">
   <property name="icon">
    <iconset resource="notepad.qrc">
     <normaloff>:/img/palette_FILL0_wght400_GRAD0_opsz48 - 副本.png</normaloff>:/img/palette_FILL0_wght400_GRAD0_opsz48 - 副本.png</iconset>
   </property>
   <property name="text">
    <string>字体(&amp;F)</string>
   </property>
   <property name="toolTip">
    <string>字体</string>
   </property>
  </action>
  <action name="actionfontcolor">
   <property name="text">
    <string>字体颜色</string>
   </property>
   <property name="toolTip">
    <string>字体颜色</string>
   </property>
  </action>
  <action name="actionfontbackgroundcolor">
   <property name="text">
    <string>字体背景色</string>
   </property>
   <property name="toolTip">
    <string>字体背景色</string>
   </property>
  </action>
  <action name="actionebackgroundcolor">
   <property name="text">
    <string>编辑器背景色</string>
   </property>
   <property name="toolTip">
    <string>编辑器背景色</string>
   </property>
  </action>
  <action name="actionShowToolBar">
   <property name="icon">
    <iconset resource="notepad.qrc">
     <normaloff>:/img/home_repair_service_FILL0_wght400_GRAD0_opsz48 - 副本.png</normaloff>:/img/home_repair_service_FILL0_wght400_GRAD0_opsz48 - 副本.png</iconset>
   </property>
   <property name="text">
    <string>工具栏(&amp;T)</string>
   </property>
   <property name="toolTip">
    <string>工具栏</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="actionShowStatusBar">
   <property name="icon">
    <iconset resource="notepad.qrc">
     <normaloff>:/img/location_home_FILL0_wght400_GRAD0_opsz48 - 副本.png</normaloff>:/img/location_home_FILL0_wght400_GRAD0_opsz48 - 副本.png</iconset>
   </property>
   <property name="text">
    <string>状态栏(&amp;S)</string>
   </property>
   <property name="toolTip">
    <string>状态栏</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionabout">
   <property name="text">
    <string>关于(&amp;A)</string>
   </property>
   <property name="toolTip">
    <string>关于</string>
   </property>
  </action>
  <action name="actionSelectAll">
   <property name="text">
    <string>全选(&amp;L)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>退出(&amp;X)</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="notepad.qrc"/>
 </resources>
 <connections/>
</ui>
